RELEASE_NOTE_DRAFT.md

```diff
@@ -31,6 +31,7 @@ The Eclipse Collections team gives a huge thank you to everyone who participated
 * Added toSortedMap and toSortedMapBy to Collectors2.
 * Added Norwegian translation for the Eclipse Collections website.
 * Added ClassComparer utility. 
+* Added peekAndPop to ImmutableStack
```

Comment:
I think this can be removed for this PR as we groom these during release process from the commits.

Reply:
Done